pkg/storagev2/chunkstore.go

```diff
@@ -17,21 +17,42 @@ type Getter interface {
 	// Get a chunk by its swarm.Address. Returns the chunk associated with
 	// the address alongside with its postage stamp, or a storage.ErrNotFound
 	// if the chunk is not found.
+	// If the chunk has multiple stamps, then the first stamp is returned in this
+	// query. In order to deterministically get the stamp, use the GetterWithStamp
+	// interface.
 	Get(context.Context, swarm.Address) (swarm.Chunk, error)
 }
 
+// GetterWithStamp is the interface that provides a way to specify a postage
+// stamp to filter the Get query. This is required as we will support multiple stamps
+// on a chunk. As a result some components would need to query chunk and provide
+// the stamp to return by using the argument.
```

Comment:
It should also be documented if it returns `storage.ErrNotFound` if the chunk is not found.